client/js/lounge.js

```diff
@@ -6,6 +6,8 @@ $(function() {
 	var path = window.location.pathname + "socket.io/";
 	var socket = io({path: path});
 	var commands = [
+		"/away",
+		"/back",
```

Comment:
I know other clients use `/back`, but I lean more towards `/unaway`. For one, `/back` is a very generic word and we might want to keep it for later. But more importantly, it's more consistent with other "toggle" actions we have: `/op` vs. `/deop`, `/voice` vs. `/devoice`, `/connect` vs. `/disconnect`, etc.
Since it's not dictacted by the protocol anyway, we have the freedom to do what we want :)